### ISSUE TITLE
Popup cut off on smaller screens (i.e. mobile)

### ISSUE THREAD

**Issue by tieroom** [husseycoding/ajaxcartpopup#28]
Hi!

Great extension!

When browsing on a small device, such as a phone with 4"-5" screen the pop-up doesn't adapt to the screen size. In my case most of the product image gets cut off. I can still see the important details and buttons so technically it doesn't stop a customer from proceeding but it does appear somewhat broken and could scare them.
![screenshot_20160413-135849](https://cloud.githubusercontent.com/assets/5046863/14493266/e0aad162-0184-11e6-95e0-f692b01f55a1.png)

More of a feature request than a bug I guess although with almost half of ecommerce traffic coming from mobile these days it's important to get it right.

I'm on an almost clean version of RWD on magento 1.9.2.2. Verified this issue is the same on a clean version.

Thanks,
Christian

**Comment by husseycoding** [comment]
Yep ok, I'll have a think about adding responsive styling to the next version.

**Comment by tieroom** [comment]
Awesome!

Probably a bunch of other things to consider for mobile while you are at it, like mouseover to produce the popup is nice on desktop but obviously only click works for mobile so a combo setting would be nice. Anywho, I'll happily beta-test when/if you release a responsive version.

**Comment by husseycoding** [comment]
Ok thanks for the suggestion.
